snap/snapcraft.yaml

```diff
@@ -4,6 +4,7 @@ adopt-info: gradle
 base: core20
 grade: stable
 confinement: classic
+compression: lzo
```

Comment:
Ain't the lzo compression has a snapd version dependency?   We should probably set the `assumes` property to ensure that systems with unsupported snapd version will not try to install the package.

Reply:
According to [Application startup performance - snapd - snapcraft.io](https://forum.snapcraft.io/t/application-startup-performance/17273/13), the first snapd version that supports this feature seems to be 2.47.1.

Reply:
LZO compression only depends on the kernel support, it's independent of snapd's version.

Reply:
Got it, thanks.